.changeset/plenty-foxes-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Fix #5620, `noConsole` rule now correctly handles indirect `console.log` calls and references.
```

Comment:
Please refer to the contribution guide regarding the changelog:
- use the past "Fixed"
- link the rule

Reply:
Sure, fixed.